video/types.h

```diff
@@ -0,0 +1,171 @@
+//+--------------------------------------------------------------------------
+//
+// File:        types.h
+//
+// Original Source info:
+// NightDriverStrip - (c) 2023 Plummer's Software LLC.  All Rights Reserved.
```

Comment:
the original source version of this file contained a few other features that we don't (and won't) need, so they have been removed